pkg/login/social/social_test.go

```diff
@@ -0,0 +1,30 @@
+package social
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Use `require` instead to make the test fail directly instead of continue as when using `assert`.
```suggestion
	"github.com/stretchr/testify/require"
```

Reply:
Updated